pages/community/precice-workshop-2024.md

```diff
@@ -31,7 +31,11 @@ Are you revisiting one of the classical preCICE use cases? Did you already prese
 
 ### Registration
 
-The registration for the workshop will open at the begining of May. You will be able to register for the full workshop or for the main part only, in case you already know the topics covered by the course or you cannot attend on the first days.
+Registration workshop is possible until September 15. Register before July 15 to benefit from early-bird prices!
+
+You can register for the full workshop or for the main part only, in case you already know the topics covered by the course or you cannot attend on the first days.
```

Comment:
```suggestion
You can register for the full workshop (talks + course) or for the main part (only talks), in case you already know the topics covered by the course or you cannot attend on the first days.
```